pom.xml

```diff
@@ -476,6 +476,11 @@
                 <version>${junit.version}</version>
                 <scope>test</scope>
             </dependency>
+            <dependency>
```

Comment:
If possible, shall we add a comment similar to this ?
https://github.com/wso2/product-microgateway/blob/main/pom.xml#L552

Reply:
Comment added